src/foam/nanos/logger/CompositeLogger.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.logger;
+
+import foam.nanos.logger.FileLogger;
+import foam.nanos.logger.StdoutLogger;
+import foam.nanos.logger.Logger;
+import java.util.ArrayList; 
+import java.util.List; 
+import java.util.logging.*;
+import java.io.IOException;
+
+public class CompositeLogger
+implements foam.nanos.logger.Logger
```

Comment:
The delegates aren't nanoservices anymore. they just have a start function. You told me that logger does not need to be a nanoservice and all of them implement logger

nothing is a nanoservice in this pull request